tools/build-base-images.sh

```diff
@@ -29,27 +29,12 @@ buildx-create
 
 HUBS="${HUBS:?specify a space seperated list of hubs}"
 TAG="${TAG:?specify a tag}"
-DOCKER_TARGETS="${DOCKER_TARGETS:-docker.base docker.distroless docker.app_sidecar_base_debian_9 docker.app_sidecar_base_debian_10 docker.app_sidecar_base_ubuntu_xenial docker.app_sidecar_base_ubuntu_bionic docker.app_sidecar_base_ubuntu_focal docker.app_sidecar_base_centos_7 docker.app_sidecar_base_centos_8}"
-
-# Verify that the specified TAG does not exist for the HUBS/TARGETS
-# Will also fail if user doesn't have authorization to repository, but they shouldn't
-# be able to push if no authorization.
-# What other errors might happen that would be ignored ?
-set +e
-for hub in ${HUBS}
-do
-  for image in ${DOCKER_TARGETS#docker.}  # assume the image name is the target without the leading docker.
-  do
-    docker manifest inspect "$hub"/"$image":"$TAG" && exit 1 # will exit if it finds the manifest
-  done
-done
-set -e
+DOCKER_TARGETS="${DOCKER_TARGETS:-base,distroless,app_sidecar_base_debian_9,app_sidecar_base_debian_10,app_sidecar_base_ubuntu_xenial,app_sidecar_base_ubuntu_bionic,app_sidecar_base_ubuntu_focal,app_sidecar_base_centos_7,app_sidecar_base_centos_8}"
 
 # For multi architecture building:
 # See https://medium.com/@artur.klauser/building-multi-architecture-docker-images-with-buildx-27d80f7e2408 for more info
 # * docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 # * docker buildx create --name multi-arch --platform linux/amd64,linux/arm64 --use
 # * export DOCKER_ARCHITECTURES="linux/amd64,linux/arm64"
 # Note: if you already have a container builder before running the qemu setup you will need to restart them
-
-BUILDX_BAKE_EXTRA_OPTIONS="--no-cache --pull" DOCKER_TARGETS="${DOCKER_TARGETS}" make dockerx.pushx
+"${ROOT}/tools/docker" --push --no-cache --no-clobber --targets="${DOCKER_TARGETS}"
```

Comment:
I think HUBS=$HUBS need to happen on this line or it has to be `export HUBS=...` above?

Reply:
Nevermind, we're reading from the environment at the beginning anyways.